Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:11.13.0-alpine
+
+# create destination directory
+RUN mkdir -p /usr/src/poc-intake
+WORKDIR /usr/src/poc-intake
+
+# update and install dependency
+RUN apk update && apk upgrade
+
+COPY . /usr/src/poc-intake/
```

Comment:
Copy package.json and lockfile first so dependencies dont need to be reinstalled on each code change

Reply:
all noted.... i'm rebuilding this file now